### DISCUSSION TITLE
In Python, I'd like to print the code chunk name

### DISCUSSION THREAD

**Discussion by bryanwhiting** [quarto-dev/quarto-cli#5714] [Feature Requests]
### Description

This isn't helpful:
```
Cell 1/20...Done
  Cell 2/20...Done
  Cell 3/20...Done
  Cell 4/20...Done
  Cell 5/20...Done
  Cell 6/20...Done
  ```

What would be helpful is for it to print the name of the code chunk, like Rmarkdown does.

**Comment by mcanouil** [comment]
I am closing here to follow-up on the associated issue (#5752).
Thanks for the suggestion.
